functions.js

```diff
@@ -1 +1,85 @@
-// YOUR CODE HERE
+// maxOrMin
+function maxOrMin(num1, num2, bool) {
+  if(bool) {
+    return (num1 > num2)?num1:num2;
+  } else if (!bool) {
+    return (num1 < num2)?num1:num2;
+  }
+}
+
+maxOrMin(6, 2, false);
+
+//sillySum(arr)
+
+var randomNumbers = [1, 5, 3, 10, 9];
+
+function sillySum(array) {
+   return array.map(function newArrayFunction(currentValue, index) {
+       return currentValue + (currentValue * index);
+   });
+}
+
```

Comment:
Close, but you don't multiply the first value by its index. Solid use of a callback, but you've got a classic off by 1 error. Congrats ;) . The first value in the resulting array should always be 0, and the second should be the same as in the original array. The problem is in the logic following the return